views/Admin.react.js

```diff
@@ -739,6 +745,30 @@ class AdminPage extends Component {
                 dataKey="access"
                 width={this.state.width / 6}
               />
+              <Column
+                label="Account Expiration"
+                cellRenderer={({ cellData, rowData: { uid } }) => (
+                  console.log({ cellData, uid }),
+                  (
+                    <TextField
+                      id="account_expiration"
+                      type="date"
+                      defaultValue={moment(cellData).format('YYYY-MM-DD')}
+                      InputLabelProps={{
+                        shrink: true,
+                      }}
+                      onChange={(e) =>
+                        this.updateAccountExpiration({
+                          uid,
+                          account_expires: e.target.value,
+                        })
+                      }
+                    />
+                  )
+                )}
+                dataKey="account_expires"
+                width={this.state.width / 4}
```

Comment:
the other columns divide the state width by 6, why does this divide by 4?

Reply:
When it is divided by 6 the calendar on the side becomes hidden. Dividing by 4 means a wonder column